sim/paladin/aura_mastery.go

```diff
@@ -0,0 +1,56 @@
+package paladin
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+	"github.com/wowsims/sod/sim/core/stats"
+)
+
+func (paladin *Paladin) registerAuraMastery() {
+
+	if !paladin.hasRune(proto.PaladinRune_RuneLegsAuraMastery) {
+		return
+	}
+
+	cd := core.Cooldown{
+		Timer:    paladin.NewTimer(),
+		Duration: time.Minute * 2,
+	}
+
+	aura := paladin.RegisterAura(core.Aura{
+		Label:    "Aura Mastery",
+		ActionID: core.ActionID{SpellID: 415756},
+		Duration: time.Second * 6,
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+		    if paladin.currentPaladinAura != nil {
+                        if paladin.currentPaladinAura.Label == "Sanctity Aura" {
+                            paladin.PseudoStats.SchoolDamageDealtMultiplier[stats.SchoolIndexHoly] *= (1.2 / 1.1)
+                        }
+                    }
+		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+                    if paladin.currentPaladinAura != nil {
+                        if paladin.currentPaladinAura.Label == "Sanctity Aura" {
+                            paladin.PseudoStats.SchoolDamageDealtMultiplier[stats.SchoolIndexHoly] /= (1.2 / 1.1)
+                        }
+                    }
+		},
+	})
```

Comment:
Looks like the indentation got a bit wonky here

Reply:
will fix